Close#2_Create_greeting_page

MrErgos opened the pull request:
1.Main controller created
1.2 Annotation @Controller is attached to the MainController
1.3 Method getHtml() created
2.Welcome page created

MrErgos committed:
#2_Create_greeting_page
1.Main controller created
1.2 Annotation @Controller is attached to the MainController
1.3 Method getHtml() created
2.Welcome page created

spasynkov commented on src/main/resources/static/WelcomePage.html:
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
```
это вообще ни разу не критично, но считается, что если язык указан английский - то и контент на странице должен быть на английском языке.
для нашего случая это вообще не важно. но вообще - браузер смотрит на это. человек в браузере может настроить приоритетность отображения языков.
у меня, например, в браузере стоит сначала украинский, потом русский, потом английский. это значит что если сервер отдает страницы на русском и английском - то мне будет показана страница на русском.
но вот если какой-то бразилец откроет нашу страницу (и если бы у нас были разные варианты для разных языков, конечно же), и у него в настройках браузера стоит бразильский и английский - то ему не понравится, что сервер ему отдаст вроде как английскую страницу, но на ней контент будет на русском :)

MrErgos replied:
Я изменил на ru

MrErgos committed:
#2_Create_greeting_page
1.Main controller created
1.2 Annotation @Controller is attached to the MainController
1.3 Method getHtml() created
2.Welcome page created
2.1 Language has changed to ru
